src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -211,7 +211,7 @@ private JsonNode getQueryMetadata(String queryID) throws SQLException {
             HttpUtil.executeGeneralRequest(
                 get,
                 loginTimeout,
-                authTimeout,
+                0,
```

Comment:
I think we should have a comment here describing why this change happened

Reply:
added